cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,24 @@
+# This uninstall script is copied from the CMake wiki page:
+# https://cmake.org/Wiki/CMake_FAQ#Can_I_do_.22make_uninstall.22_with_CMake.3F
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
```

Comment:
This will break if any of the paths in `${files}` contains a newline. While none of the paths in Chimera contain newlines, this can occur if `CMAKE_CURRENT_BINARY_DIR` contains a newline. I would prefer to delimit `install_manifest.txt` by `NUL` characters, which does not suffer from the same ambiguity.